runtime/src/accounts_db.rs

```diff
@@ -2195,4 +2225,78 @@ pub mod tests {
             "Account-based hashing must be consistent with StoredAccount-based one."
         );
     }
+
+    #[test]
+    fn test_verify_bank_hash() {
```

Comment:
As the coverage report from the CI will reveal, these tests does the 100% coverage. :p